.github/actions/bouf/action.yaml

```diff
@@ -0,0 +1,127 @@
+name: Run bouf
+description: Generates signed OBS install files and updater files
+inputs:
+  updaterPrivateKey:
+    description: Private key for signing update manifest
+    required: true
+  gcpWorkloadIdentityProvider:
+    description: GCP Identity Provider Pool ID
+    required: true
+  serviceAccountName:
+    description: Google service account name
+    required: true
+  version:
+    description: Version string (e.g. 30.0.0-rc1)
+    required: true
+  channel:
+    description: Update channel
+    required: false
+    default: 'stable'
+  commitHash:
+    description: Commit hash
+    required: false
+    default: ''
+
+runs:
+  using: composite
+  steps:
+    - name: Extract Artifact
+      shell: pwsh
+      run: |
+        Expand-Archive -Path build\*.zip -DestinationPath build
+        Remove-Item build\*.zip
```

Comment:
Did the asterisks actually need to be escaped?

Reply:
I think it's matching files in the `build` subdirectory.

Reply:
> I think it's matching files in the `build` subdirectory.

Oh, that makes more sense.

Reply:
Yep, remember Windows supports both `/` and `\` but `\` is technically correct (and sometimes built-in stuff won't work correctly with `/`)